src/server/utils/util.ts

```diff
@@ -0,0 +1,50 @@
+import { parse, getYear, getMonth, format, parseISO } from 'date-fns';
+import DateTimeFormatOptions = Intl.DateTimeFormatOptions;
+export const datoFormat: DateTimeFormatOptions = {
+  day: '2-digit',
+  month: '2-digit',
+  year: 'numeric',
+};
```

Comment:
Hvorfor har vi to utilsmapper? Jeg ser at nøyaktig den samme metoden blir definert i `src/utils/dato.ts`. Her burde vi ta ansvar og rydde opp slik at vi ikke definerer dobbelt opp med metoder rundt omkring

Reply:
Dette gjelder også andre metoder som for eksempel `formaterIsoDato`